docker/test/integration/features/s3.feature

```diff
@@ -36,14 +36,14 @@ Feature: Sending data from MiNiFi-C++ to an AWS server
     And the "success" relationship of the GetFile processor is connected to the PutS3Object
     And the "success" relationship of the PutS3Object processor is connected to the PutFile
 
-    And a s3 server "s3" is set up in correspondence with the PutS3Object
-    And the http proxy server "http-proxy" is set up
+    And a s3 server is set up in correspondence with the PutS3Object
+    And the http proxy server is set up
     When all instances start up
 
     Then a flowfile with the content "test" is placed in the monitored directory in less than 150 seconds
-    And the object on the "s3" s3 server is "LH_O#L|FD<FASD{FO#@$#$%^ \"#\"$L%:\"@#$L\":test_data#$#%#$%?{\"F{"
-    And the object content type on the "s3" s3 server is "application/octet-stream" and the object metadata matches use metadata
-    And no errors were generated on the "http-proxy" regarding "http://s3-server:9090/test_bucket/test_object_key"
+    And the object on the s3 server is "LH_O#L|FD<FASD{FO#@$#$%^ \"#\"$L%:\"@#$L\":test_data#$#%#$%?{\"F{"
+    And the object content type on the s3 server is "application/octet-stream" and the object metadata matches use metadata
+    And no errors were generated on the http-proxy regarding "http://s3-server:9090/test_bucket/test_object_key"
```

Comment:
Just a question: What's the significance of this step?

Reply:
We check that in the proxy server logs that the call was actually done through the proxy server without a problem thus the proxy configuration works.

Reply:
Ahan okay. Thanks :)